Kudu.Core/Jobs/TriggeredJobsScheduler.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Kudu.Contracts.Jobs;
+using Kudu.Core.Hooks;
+using Kudu.Core.Infrastructure;
+using Kudu.Core.Tracing;
+
+namespace Kudu.Core.Jobs
+{
+    public class TriggeredJobsScheduler : IDisposable
```

Comment:
A schedule is an added property on top of a triggered job so I think it makes sense as it is.